raiden-dapp/src/components/navigation/Transfer.vue

```diff
@@ -1,55 +1,74 @@
 <template>
   <v-container fluid class="transfer">
-    <v-row justify="center" no-gutters>
-      <amount-display
-        class="transfer__token-network-amount"
-        exact-amount
-        :amount="capacity"
-        :token="token"
-      />
+    <div class="transfer__menus">
+      <v-row no-gutters class="transfer__menus__items">
+        <v-col cols="11">
+          <div class="transfer__menus__items__token-select">
+            <span @click="showTokenNetworks = true">
+              {{ $t('transfer.change-token-title') }}
+              <v-icon>mdi-chevron-down</v-icon>
+            </span>
+          </div>
+        </v-col>
+        <v-col cols="1">
+          <v-menu transition="scale-transition">
+            <template #activator="{ on }">
+              <v-btn icon v-on="on">
+                <v-icon large>mdi-dots-vertical</v-icon>
+              </v-btn>
+            </template>
+            <div class="transfer__menus__items__channels-deposit">
+              <v-row justify="center">
+                <span
+                  class="transfer__menus__items__channels-deposit--deposit"
+                  @click="depositing = true"
+                >
+                  {{ $t('transfer.deposit-button') }}
+                </span>
+              </v-row>
+              <v-row justify="center">
+                <span
+                  class="transfer__menus__items__channels-deposit--channels"
```

Comment:
Having that long classes in BIM notation like here shows perfectly that this component is quite nested. So for me this is too nested. I loose the overview. Maybe worth to split into smaller chunks on a later issue. What do you think? Worth to open an issue to improve quality? I like index directory components, but not sure what the general opinion on that is.

Reply:
Definitely worth bringing up for discussion.

Reply:
I created [this issue](https://github.com/raiden-network/light-client/issues/1888). So it doesn't get lost and someday we maybe can tackle this.